package.json

```diff
@@ -93,6 +93,8 @@
     "jest": "^20.0.4",
     "mkdirp": "^0.5.1",
     "ncp": "^2.0.0",
+    "node-sass": "^4.7.2",
+    "sass-loader": "^6.0.6",
```

Comment:
Is this needed as a top level dev dep?

Reply:
DevDep, yes. That way we can skip installation _during_ the tests. Moving it here allows it to be cached, doesn't reinstall all other devdeps, and was a big player in speeding up the `build` test from minutes to seconds.

Reply:
Alright, makes sense ig